main/3  EDU/Tab.16.2.1.sql

```diff
@@ -74,36 +74,36 @@ CFF.CODUL    AS NR_SECTIE,
   CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN (1058)  THEN D.COL1 END)) AS COL1,
   CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL2 END)) AS COL2,
  
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL12 END)) -
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL13 END)) AS COL3,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL1 END)) -
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL2 END)) AS COL3,
   
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL12 END)) -
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL14 END)) AS COL4,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL1 END)) -
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL3 END)) AS COL4,
   
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL14 END)) AS COL5,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  THEN D.COL3 END)) AS COL5,
      
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL12 END)) AS COL6,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL1 END)) AS COL6,
   
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL13 END)) AS COL7,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL2 END)) AS COL7,
  
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL12 END))- 
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL13 END)) AS COL8,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL1 END))- 
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL2 END)) AS COL8,
   
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL12 END))-
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL14 END)) AS COL9,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL1 END))-
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (1) THEN D.COL3 END)) AS COL9,
   
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (1) THEN D.COL14 END)) AS COL10,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (1) THEN D.COL3 END)) AS COL10,
     
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL12 END)) AS COL11,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL1 END)) AS COL11,
     
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL13 END)) AS COL12,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL2 END)) AS COL12,
       
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (25)  THEN D.COL12 END))-
-  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL13 END)) AS COL13,
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058  AND R.NFI IN (25)  THEN D.COL1 END))-
+  CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL2 END)) AS COL13,
```

Comment:
CIS2.NVAL(SUM(CASE WHEN D.CAPITOL IN 1058   AND R.NFI IN (25) THEN D.COL2 END)) AS COL13,